app/start_handlers.py

```diff
@@ -146,10 +147,9 @@ async def post(self, request):
                     },
                     status=401)
             else:
+                logger.error('invalid access code', client_ip=request['client_ip'])
```

Comment:
Log text needs to be something like 'error processing access code' - this error is triggered when RHSvc returns any error not triggered by an invalid code